Entities/Actor.h

```diff
@@ -583,17 +576,15 @@ ClassInfoGetters
 // Arguments:       An int specifying the value to add.
 // Return value:    The resulting total health of this Actor.
 
-    const int AddHealth(const int addedHealth) { return m_Health += addedHealth; }
-
+    const float AddHealth(const float addedHealth) { return m_Health += addedHealth; }
```

Comment:
There's probably no reason to make the input params const for AddHealth and SetHealth

Also, you forgot to fix the method comment for AddHealth while you were poking at it.